core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/bundle/BundleRequests.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 B2i Healthcare Pte Ltd, http://b2i.sg
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.b2international.snowowl.core.bundle;
+
+/**
+ * @since 8.0
+ */
+public final class BundleRequests {
```

Comment:
Convert this to an instantiatable class and add a new method `ResourceRequests.bundles()` that creates it, so you can have an API like this:
```
ResourceRequests.bundles().prepareCreate()...
ResourceRequests.bundles().prepareSearch()...
ResourceRequests.bundles().prepareGet(id)...
ResourceRequests.bundles().prepareDelete(id)...
ResourceRequests.bundles().prepareUpdate(id)...
```